### PR TITLE
Allow all content items to be assigned on a page

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

**Comment by andrewgarner** [comment]
[(3) Allocate all content items](https://trello.com/c/B2byRQkl/428-3-allocate-all-content-items)

File: spec/features/audit/allocation_spec.rb [hunk 1]
```diff
@@ -104,4 +104,19 @@
     expect(page).to have_select("allocate_to", selected: "No one")
     expect(page).to have_content("1 items unallocated")
   end
+
+  scenario 'Allocate all content within current page', :js do
+    create_list(:content_item, 100)
```
Comment (pmanrubia) [inline spec/features/audit/allocation_spec.rb]: Creating a list of 100 seems a bit too much. What about creating 27 items (page size + a couple of them more)?

File: app/assets/javascripts/select-all.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+(function (Modules) {
+  "use strict";
+
+  Modules.SelectAll = function () {
+    var that = this;
+
+    that.start = function (element) {
```
Comment (pmanrubia) [inline app/assets/javascripts/select-all.js]: I would say that we should use `this` at this point, and use `that` inside the event handler.
But, as we are not using the reference to `this` in lines 13-15 we could get rid of `that`

Comment (chao-xian) [inline app/assets/javascripts/select-all.js]: Addressed in https://github.com/alphagov/content-performance-manager/pull/290/commits/d4fd166a4ef59282c85bf2d178a0748e0c77dbf3#diff-1bf7091d9f272b27a4976caf9081ac8cR5

File: spec/features/audit/allocation_spec.rb (hunk 1 above)
Comment (chao-xian) [inline spec/features/audit/allocation_spec.rb]: Addressed in https://github.com/alphagov/content-performance-manager/pull/290/commits/d4fd166a4ef59282c85bf2d178a0748e0c77dbf3#diff-69c737006735f69541cec3ef0afaea49R109

**Pull request merged**